torch/onnx/symbolic_opset18.py

```diff
@@ -0,0 +1,54 @@
+"""This file exports ONNX ops for opset 18.
+
+Note [ONNX Operators that are added/updated in opset 18]
+
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+https://github.com/onnx/onnx/blob/main/docs/Changelog.md#version-18-of-the-default-onnx-operator-set
+New operators:
+    CenterCropPad
+    Col2Im
+    Mish
+    OptionalGetElement
+    OptionalHasElement
+    Pad
+    Resize
+    ScatterElements
+    ScatterND
+"""
+
+from typing import Sequence
+
+from torch import _C
+from torch.onnx import symbolic_helper
+from torch.onnx._internal import _beartype
+
+
+# EDITING THIS FILE? READ THIS FIRST!
+# see Note [Edit Symbolic Files] in symbolic_helper.py
+
```

Comment:
We need to add col2im to `__all__`

Reply:
Isn't `__all__` automatically populated? We only customize it when we want to remove something from it. 
Which change caused this extra step to be needed? It is hard to keep static lists up to date

Reply:
unfortunately this is how pytorch maintainer suggested https://github.com/pytorch/pytorch/wiki/Public-API-definition-and-documentation. We needed all because we are also importing from typing (which we want to do) and the classes from typing "appears" to be public

Reply:
After adding this file to init I would expect a test error that says this

Reply:
this will be temporary though. After we take the symbolic functions into internal we will have a programmatic way of populating all